cypress/integration/admin/pageBuilder/page/importExportPages.spec.js

```diff
@@ -1,129 +1,133 @@
-//         cy.findByTestId("import-pages.loading-dialog").within(() => {
-//             cy.findByText("All pages have been imported").should("be.visible");
-//             cy.findByText(/Show details/i).click();
-//             cy.findByTestId("import-pages-dialog.show-detail-list")
-//                 .children()
-//                 .should("have.length", 1);
-//
-//             cy.findByText(/Continue/i).click();
-//         });
-//
-//         // Page should be there
-//         searchForPage(pageTitle);
-//
-//         cy.findByTestId("default-data-list").within(() => {
-//             cy.get(".mdc-list-item")
-//                 .first()
-//                 .within(() => {
-//                     cy.findByText("Welcome to Webiny").should("exist");
-//                     cy.findByText(/Static/i).should("exist");
-//                     cy.findByText(/Draft/i).should("exist");
-//                     cy.findByText(/(v1)/i).should("exist");
-//                     cy.findByTestId("pages-default-data-list.select-page").click({ force: true });
-//                 });
-//         });
-//
-//         // Delete the imported page
-//         cy.findByTestId("default-data-list").within(() => {
-//             cy.get(".mdc-list-item")
-//                 .first()
-//                 .within(() => {
-//                     cy.findByTestId("pages-default-data-list.select-page").click({ force: true });
-//                 });
-//         });
-//         cy.findByTestId("pb-page-details-header-delete-button").click();
-//         cy.findByTestId("pb-page-details-header-delete-dialog").within(() => {
-//             cy.findByText(/Confirm/i).click();
-//         });
-//         cy.findByTestId("default-data-list").within(() => {
-//             cy.get(".mdc-list-item")
-//                 .first()
-//                 .within(() => {
-//                     cy.findByText(/Draft/i).should("not.exist");
-//                 });
-//         });
-//     });
-// });
+// We need this to grant Cypress access to the clipboard.
+Cypress.automation("remote:debugger:protocol", {
+    command: "Browser.grantPermissions",
+    params: {
+        permissions: ["clipboardReadWrite", "clipboardSanitizedWrite"],
+        origin: window.location.origin
+    }
+});
+
+context("Export & Import Pages", () => {
+    beforeEach(() => cy.login());
+
+    describe("When exporting and importing page", () => {
+        const id = uniqid();
+        const pageTitle1 = `Test published page 1`;
+
+        beforeEach(() => {
+            cy.pbCreatePage({ category: "static" }).then(page => {
+                cy.pbUpdatePage({
+                    id: page.id,
+                    data: {
+                        category: "static",
+                        path: `/page-${id}`,
+                        title: pageTitle1
+                    }
+                });
+                cy.pbPublishPage({ id: page.id });
+            });
+        });
+
+        afterEach(() => {
```

Comment:
Looks good, although let's rather have the delete all pages step as the first thing we do in the `beforeEach` block.

The cool thing is that `cy.pbDeleteAllPages` command already exists, but that code wasn't yet merged.

Feel free to c/p stuff from [here](https://github.com/webiny/webiny-js/pull/2582/files#diff-17c566c0b0c88cdf359f34698d2e32f0444f8e927abdf600f00ac2409ced876e).

Usage example [here](https://github.com/webiny/webiny-js/pull/2582/files#diff-4c62b28a2ca1d93a725cf7276df1453d9bbcd15363820cde9e64cf2f8299e45d).

Sorry for the last-minute change, I thought this was already merged, but looks like it wasn't.